charts/myriad-api/templates/deployment.yaml

```diff
@@ -140,6 +144,14 @@ spec:
                   name: {{ include "myriad-api.coinMarketCapSecretName" . }}
                   key: apiKey
             {{- end }}
+            {{- if .Values.config.openApiDisable }}
```

Comment:
please move this before MYRIAD_RPC_WS_URL

Reply:
done